tests/lib/rules/block-spacing.js

```diff
@@ -240,103 +299,275 @@ ruleTester.run("block-spacing", rule, {
             output: "{foo();}",
             options: ["never"],
             errors: [
-                { type: "BlockStatement", line: 1, column: 1, messageId: "extra", data: { location: "after", token: "{" } },
-                { type: "BlockStatement", line: 1, column: 10, messageId: "extra", data: { location: "before", token: "}" } }
+                {
+                    type: "BlockStatement",
+                    messageId: "extra",
+                    data: { location: "after", token: "{" },
+                    line: 1,
+                    column: 2,
+                    endLine: 1,
+                    endColumn: 3
+                },
+                {
+                    type: "BlockStatement",
+                    messageId: "extra",
+                    data: { location: "before", token: "}" },
+                    endLine: 1,
+                    endColumn: 10,
+                    line: 1,
+                    column: 9
```

Comment:
`line` and `column` might be before `endLine` and `endColumn`, just for consistency with other tests.